### ISSUE TITLE
Distribute using 7z

### ISSUE THREAD

**Issue by develar** [electron/electron#5455]
Currently, you use zip.

File [electron-v0.37.8-win32-x64.zip]( https://github.com/electron/electron/releases/download/v0.37.8/electron-v0.37.8-win32-x64.zip)

zip: 47M
7z :  30M

difference: **44%  better** (using `-m0=lzma2 -mx=9 -mfb=64 -md=32m -ms=on`).

Doesn't make sense on user machine (installed only once). 
But it worth on CI servers. Not all users caches `.electron` (electron-download cache dir). Travis until last week didn't  support cache on OS X. And in any case is not cached if build failed or new forked repo.

Since your releases contains `electron-v0.37.8-darwin-x64-dsym.zip` — 1.39 GB, disk space is cheap for you and you can publish electron in the 7z format (using `-m0=lzma2 -mx=9 -mfb=64 -md=32m -ms=on`) :) I hope :)

I successfully use 7z in project [electron-builder](https://github.com/electron-userland/electron-builder), there are 7za binaries for OS X, Windows and Linux to use it in the node without any native deps (https://www.npmjs.com/package/7zip-bin). So, it is not a problem to extract electron in 7z using nodejs.

About `xz` — slower and worse. And it is not easy to extract it using nodejs (7za - one command, in case of `xz` you have to use tar&xz).

**Comment by MarshallOfSound** [comment]
@develar All platforms normally by default include UNZIP functionality, very few by default support 7zip.  I think it comes down to usability vs file size, on my CI builds downloading electron releases takes maybe 3 seconds whereas building takes 8-10 minutes.

Not saying I wouldn't appreciate having a smaller `.electron` folder but I don't see this as great usability choice or a massive improvement (these file sizes only affect developers).

**IMO**

**Comment by develar** [comment]
> All platforms normally by default include UNZIP functionality,

And `xz` (OS X since Mac OS X 10.10 “Yosemite”), windows as always doesn't. But xz is  slower and worse (35M, compression time x3).

> I don't see this as great usability choice or a massive improvement 

Yes. But since there is 1.39 GB file in the artifacts, maybe electron team will don't mind to distribute in 7z as well. Because it is just several lines in the publish script.

**Comment by zcbenz** [comment]
We already have so many electron tools handling zip distributions, changing to use 7z would break them all. I'm closing this as won't fix.

**Comment by develar** [comment]
> changing to use 7z

Not changing, but in addition to zip. e.g. ruby can be downloaded as `tar.gz` (default), or as `tar.xz`.
